replication/row_event.go

```diff
@@ -20,42 +20,46 @@ import (
 var errMissingTableMapEvent = errors.New("invalid table id, no corresponding table map event")
 
 type TableMapEvent struct {
-	flavor      string
-	tableIDSize int
+	/*
+	   The some fields are available only after MySQL-8.0.1 or MariaDB-10.5.0
```

Comment:
I don't like that before we had a clear separation between the "old" and "new" fields. And now they're all mixed up. :(

But I have no idea how to fix this w/o nolint. Do you have some ideas maybe?

Reply:
Manually aligning the fields is a real puzzle, and probably you wouldn't end up with the perfect readable order after all. I'll just add nolint to it